CHANGELOG.unreleased.md

```diff
@@ -8,6 +8,7 @@
 
 - [Settings/Servers] Fix read-only setting toggling
 - [SDN Controller] Do not choose physical PIF without IP configuration for tunnels. (PR [#4319](https://github.com/vatesfr/xen-orchestra/pull/4319))
+- [SDN Controller] Better monitor of hosts' lifecycle. (PR [#4314](https://github.com/vatesfr/xen-orchestra/pull/4314))
```

Comment:
What does it fix?

Reply:
It used to monitor hosts' state by using `host.enabled` instead of `host.$metrics.live` therefore the detection of host shutting down or being restarted was badly detected.

Reply:
IMHO, this is what we should put in the changelog, so users can say "this is the issue I had, glad it's fixed".